datahub-web-react/src/app/entity/chart/chartMatchedFieldRenderer.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
```

Comment:
This file is super confusing. I just ported it from the snippet component that was here.